### DISCUSSION TITLE
[PSD12] Exercise 3.12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#180] [Esercizi (A-L)]
**Exercise 3.12** Design an asynchronously resettable D latch using logic gates.

**Comment by Elia-Belli** [comment]
**Edit - Piccolo Errore nel Disegno**: quando $R$ entra nell' $AND3$ deve essere negato, infatti la funzione è $S = CD\bar R$, quindi per il circuito fate riferimento al post di @fraxb04 

Un D-Latch è costruito a partire da un SR-Latch e della logica combinatoria, modificando la logica combinatoria ho inserito un segnale R che resetta il Latch in modalità asincrona.
![ARDL](https://user-images.githubusercontent.com/107944476/211785235-2ba5d5d3-136b-4428-a83c-494a0c998413.jpg)

**Comment by fraxb04** [comment]
@Elia-Belli 
errato, manca un NOT:
![image](https://user-images.githubusercontent.com/122573871/214033078-2320479a-b385-4b8a-9f8f-cf64fc2eaa3b.png)
